jrnl/Journal.py

```diff
@@ -434,6 +434,10 @@ def open_journal(journal_name, config, legacy=False):
     if not config["encrypt"]:
         if legacy:
             return LegacyJournal(journal_name, **config).open()
+        if config["journal"].endswith("/") or config["journal"].endswith("\\"):
```

Comment:
Could you use [os.sep](https://docs.python.org/3.9/library/os.html?highlight=os.sep#os.sep) here instead of both conditions? This will keep POSIX systems from accidentally creating folder paths ending with a backslash.